io.github.electronstudio.WeylusCommunityEdition.json

```diff
@@ -0,0 +1,87 @@
+{
+  "id": "io.github.electronstudio.WeylusCommunityEdition",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.node18"
+  ],
+  "command": "weylus",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--device=all",
+    "--share=network",
+    "--filesystem=xdg-run/pipewire-0"
+  ],
+  "modules": [
+    {
+      "name": "x264",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --prefix=/app --exec-prefix=/app --disable-opencl --disable-cli --enable-static --enable-shared",
+        "make -j$(nproc)",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://code.videolan.org/videolan/x264/-/archive/4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1/x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1.tar.bz2",
+          "sha512": "6d0d9e079d6c7650abb5c00ad60cf3cfe72a220b3ee7cd030d4daeefdd9feeb4d056cf2e01b2f8d2fb0a66ccc15cdde860237f8ac5eac42ede6e5444a81346f8"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --prefix=/app --disable-debug --disable-programs --enable-gpl --enable-libx264  --disable-autodetect --enable-static --enable-shared",
+        "make -j$(nproc)",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.ffmpeg.org/releases/ffmpeg-6.1.1.tar.xz",
+          "sha512": "fca3f8635f29182e3ae0fe843a8a53614e4b47e22c11508df3ff7cdbafbb4b5ee0d82d9b3332871f7c1032033b1cad2f67557d7c5f7f7d85e2adadca122965d5"
+        }
+      ]
+    },
+    {
+      "name": "WeylusCommunityEdition",
+      "buildsystem": "simple",
+      "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/node18/bin",
+        "env": {
+          "RUSTFLAGS": "-A dead_code -A unused_imports -L /app/lib",
+          "CARGO_HOME": "/run/build/WeylusCommunityEdition/cargo",
+          "XDG_CACHE_HOME": "/run/build/WeylusCommunityEdition/flatpak-node/cache",
+          "npm_config_cache": "/run/build/WeylusCommunityEdition/flatpak-node/npm-cache",
+          "npm_config_nodedir": "/usr/lib/sdk/node18"
+        }
+      },
+      "build-commands": [
+        "cargo b --features ffmpeg-system --release",
```

Comment:
this will probably fail because it's not offline.

Reply:
The crates are all already downloaded based on `generated-sources-cargo.json` and cargo is configured to them by CARGO_HOME, so I don't see why?  Also it works when I test it and I don't have --share=network in the build, so that's the same as running on the build machine?  But sure I can set CARGO_NET_OFFLINE if you want.

Reply:
if you say it works. we'll know soon enough.